src/main/java/krona/task/Event.java

```diff
@@ -0,0 +1,49 @@
+package krona.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Represents an Event with a timeline from a start date
+ * and time to an end date and time.
+ */
+public class Event extends Task {
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("d/M/yyyy HHmm");
```

Comment:
Since this variable, formatter, is also final and thus a constant, maybe you should change its name to FORMATTER as well